example/examples/CachedMap.js

```diff
@@ -66,8 +66,8 @@ class CachedMap extends React.Component {
                 cacheEnabled={this.state.cache}
                 zoomEnabled
                 scrollingEnabled
-                loadingIndicatorColor={"#666666"}
-                loadingBackgroundColor={"#eeeeee"}
+                loadingIndicatorColor={'#666666'}
+                loadingBackgroundColor={'#eeeeee'}
               >
```

Comment:
I appreciate your advise. I changed back to double quotes and removed curly braces